src/main/java/edu/harvard/iq/dataverse/dataaccess/SwiftAccessIO.java

```diff
@@ -697,7 +696,17 @@ Account authenticateWithSwift(String swiftEndPoint) throws IOException {
                         .setPassword(swiftEndPointSecretKey)
                         .setAuthUrl(swiftEndPointAuthUrl)
                         .createAccount();
-            } else { // assume BASIC
+            } else if (swiftEndPointAuthMethod.equals("keystone_v3")) {
+                System.out.println("using keystone_v3");
```

Comment:
Instead of `System.out.println` this should be `logger.fine`. Please see http://guides.dataverse.org/en/4.8.5/developers/coding-style.html#logging

Reply:
Oops, debugging :)